scripts/release_ppa.sh

```diff
@@ -73,11 +73,11 @@ then
     SMTDEPENDENCY=""
```

Comment:
Please update the documentation at the start of this file, and, if needed the release checklist.